pkg/cmd/utils.go

```diff
@@ -98,6 +98,14 @@ func PrepareRequest(cmd *cobra.Command, engine string) (*Request, error) {
 	openAIClient.NTokens = nTokens
 	openAIClient.NCompletions = nCompletions
 
+	if cmd.Name() == COMMAND_EDIT {
```

Comment:
Don't do this, just set `openAIClient.Engine` to the `engine` string we passed in.